src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -490,245 +450,235 @@
+              Grid.Column="1"
+              Visibility="{x:Bind local:GraphingCalculator.ShouldDisplayPanel(IsSmallState, SwitchModeToggleButton.IsOn, x:False), Mode=OneWay}">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="*"/>
+                <RowDefinition Height="2.2*" MaxHeight="400"/>
+            </Grid.RowDefinitions>
 
-                    <!-- Ideally the KeyGraphFeaturesPanel should be a frame so that navigation to and from the panel could be handled nicely -->
-                    <local:KeyGraphFeaturesPanel x:Name="KeyGraphFeaturesControl"
-                                                 Grid.RowSpan="2"
-                                                 Grid.ColumnSpan="2"
-                                                 Margin="0,4,0,0"
-                                                 KeyGraphFeaturesClosed="OnKeyGraphFeaturesClosed"
-                                                 ViewModel="{x:Bind ViewModel.SelectedEquation, Mode=OneWay}"
-                                                 Visibility="{x:Bind IsKeyGraphFeaturesVisible, Mode=OneWay}"
-                                                 x:Load="{x:Bind IsKeyGraphFeaturesVisible, Mode=OneWay}"/>
+            <!-- Ideally the KeyGraphFeaturesPanel should be a frame so that navigation to and from the panel could be handled nicely -->
+            <local:KeyGraphFeaturesPanel x:Name="KeyGraphFeaturesControl"
+                                         Grid.RowSpan="2"
+                                         Margin="0,4,0,0"
+                                         KeyGraphFeaturesClosed="OnKeyGraphFeaturesClosed"
+                                         ViewModel="{x:Bind ViewModel.SelectedEquation, Mode=OneWay}"
+                                         Visibility="{x:Bind IsKeyGraphFeaturesVisible, Mode=OneWay}"
+                                         x:Load="{x:Bind IsKeyGraphFeaturesVisible, Mode=OneWay}"/>
 
             <!-- This control should be within a grid that limits the height to keep the sticky footer functionality from breaking -->
             <local:EquationInputArea x:Name="EquationInputAreaControl"
                                      Grid.ColumnSpan="2"
                                      Margin="0,4,0,0"
                                      EquationFormatRequested="OnEquationFormatRequested"
                                      Equations="{x:Bind ViewModel.Equations}"
+                                     IsMatchAppTheme="{x:Bind IsMatchAppTheme, Mode=OneWay}"
                                      KeyGraphFeaturesRequested="OnEquationKeyGraphFeaturesRequested"
                                      Variables="{x:Bind ViewModel.Variables}"
-                                     Visibility="{x:Bind IsKeyGraphFeaturesVisible, Converter={StaticResource BooleanToVisibilityNegationConverter}, Mode=OneWay}"
-                                     IsMatchAppTheme="{x:Bind IsMatchAppTheme, Mode=OneWay}"/>
```

Comment:
You need to add this back so that dark theme still works

Reply:
Nothing was removed, only the order was changed (Xaml Styler)